tests/ui/pages/content.py

```diff
@@ -0,0 +1,50 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+from .base import Page
+from ..regions.base import Region
+
+
+class Content(Page):
+    URL_TEMPLATE = "/books/{book_slug}/pages/{page_slug}"
+
+    _main_content_locator = (
+        By.CSS_SELECTOR,
+        '#main-content > div > div > div h1[data-type="document-title"]',
+    )
+    _table_of_contents_button_locator = (By.CSS_SELECTOR, "#root > div > div > button")
+    _table_of_contents_nav_locator = (By.CSS_SELECTOR, "#root > div > div > div > div > nav")
+
+    @property
+    def loaded(self):
+        return self.find_element(*self._main_content_locator)
```

Comment:
we should really use the js hook that the app provides, that is the canonical way that we're identifying when the page is done doing _stuff_

Reply:
when we run these tests against a built environment the page will be complete before the js is even loaded, so then this code would thought the page was done loading but all interactions would fail

Reply:
Could you provide me with some js code that would allow me to resolve the js hook. It would look something like this.

https://github.com/openstax/cnx-automation/blob/try-applitools/tests/webview/ui/test_applitools.py#L25-L27

I have no idea what the return on something like that is.. i would need to experiment.